node-data/src/encoding.rs

```diff
@@ -83,6 +83,7 @@ impl Serializable for Transaction {
 impl Serializable for SpentTransaction {
     fn write<W: Write>(&self, w: &mut W) -> io::Result<()> {
         self.inner.write(w)?;
+        w.write_all(&self.block_hash)?;
```

Comment:
The addition increases the footprint of each transaction serialized in rocksdb with another 32 bytes. In order to reduce it, could we consider `write_all` the height (8 bytes) instead?